Sources/OpenAPIRuntime/Interface/HTTPBody.swift

```diff
@@ -122,24 +122,16 @@ public final class HTTPBody: @unchecked Sendable {
     public typealias ByteChunk = ArraySlice<UInt8>
 
     /// Describes how many times the provided sequence can be iterated.
-    public enum IterationBehavior: Sendable {
-
-        /// The input sequence can only be iterated once.
-        ///
-        /// If a retry or a redirect is encountered, fail the call with
-        /// a descriptive error.
-        case single
-
-        /// The input sequence can be iterated multiple times.
-        ///
-        /// Supports retries and redirects, as a new iterator is created each
-        /// time.
-        case multiple
-    }
-
-    /// The body's iteration behavior, which controls how many times
+    @available(
+        *,
+        deprecated,
+        renamed: "IterationBehavior",
+        message: "Use the top level IterationBehavior directly instead of HTTPBody.IterationBehavior."
+    ) public typealias IterationBehavior = OpenAPIRuntime.IterationBehavior
```

Comment:
Can we move this to Deprecated.swift so it doesn't get missed in the 1.0.0 sweep?

Reply:
Moved. FWIW, I always also search for the word `deprecated` in the project as some deprecations (e.g. stored properties) cannot be moved out of their primary location.